bedrock/firefox/views.py

```diff
@@ -370,6 +380,15 @@ def show_36_whatsnew_tour(oldversion):
     return oldversion < Version('36.0')
 
 
+def show_38_0_5_firstrun(version):
+    try:
+        version = Version(version)
+    except ValueError:
+        return False
+
+    return version >= Version('38.0.5') and version < Version('39.0')
```

Comment:
AFAIK, the `/firstrun` experience for 39.0 is still unknown. We may be using this template, or we may show a new template with Fx Accounts embedded. Depends on FxA changes making it in prior to 39.0.

Probably better to err on the side of caution and let this template be shown for 39.0. If a new template is needed for 39.0, we'll update the logic with that PR.